e2e-tests/gef/cypress/e2e/facility/unissued-facilities/MIN/about-unissued-facility.spec.js

```diff
@@ -0,0 +1,234 @@
+      errorSummary().contains('Cover start date cannot be before the issue date');
+    });
+
+    it('displays error messages when cover start date is over 3 months after submission date', () => {
+      applicationPreview.unissuedFacilitiesReviewLink().click();
+      unissuedFacilityTable.updateIndividualFacilityButton(0).click();
+
+      cy.keyboardInput(aboutFacilityUnissued.coverStartDateDay().clear(), dateConstants.threeMonthsOneDayDay);
+      cy.keyboardInput(aboutFacilityUnissued.coverStartDateMonth().clear(), dateConstants.threeMonthsOneDayMonth);
+      cy.keyboardInput(aboutFacilityUnissued.coverStartDateYear().clear(), dateConstants.threeMonthsOneDayYear);
+      cy.clickContinueButton();
+
+      aboutFacilityUnissued.coverStartDateError().contains('The cover start date must be within 3 months of the inclusion notice submission date');
+      errorSummary().contains('The cover start date must be within 3 months of the inclusion notice submission date');
+    });
+
+    it('displays error messages when cover end date is before the cover start date', () => {
+      applicationPreview.unissuedFacilitiesReviewLink().click();
+      unissuedFacilityTable.updateIndividualFacilityButton(0).click();
+
+      cy.keyboardInput(aboutFacilityUnissued.coverStartDateDay().clear(), dateConstants.twoMonthsDay);
+      cy.keyboardInput(aboutFacilityUnissued.coverStartDateMonth().clear(), dateConstants.twoMonthsMonth);
+      cy.keyboardInput(aboutFacilityUnissued.coverStartDateYear().clear(), dateConstants.twoMonthsYear);
+      cy.keyboardInput(aboutFacilityUnissued.coverEndDateDay().clear(), dateConstants.twentyEightDay);
+      cy.keyboardInput(aboutFacilityUnissued.coverEndDateMonth().clear(), dateConstants.twentyEightMonth);
+      cy.keyboardInput(aboutFacilityUnissued.coverEndDateYear().clear(), dateConstants.twentyEightYear);
+
+      cy.clickContinueButton();
+
+      aboutFacilityUnissued.coverEndDateError().contains('Cover end date cannot be before cover start date');
+      errorSummary().contains('Cover end date cannot be before cover start date');
+    });
+
+    it('displays error messages when cover end date is same as the cover start date', () => {
+      applicationPreview.unissuedFacilitiesReviewLink().click();
+      unissuedFacilityTable.updateIndividualFacilityButton(0).click();
+
+      cy.keyboardInput(aboutFacilityUnissued.coverStartDateDay().clear(), dateConstants.todayDay);
+      cy.keyboardInput(aboutFacilityUnissued.coverStartDateMonth().clear(), dateConstants.todayMonth);
+      cy.keyboardInput(aboutFacilityUnissued.coverStartDateYear().clear(), dateConstants.todayYear);
+      cy.keyboardInput(aboutFacilityUnissued.coverEndDateDay().clear(), dateConstants.todayDay);
+      cy.keyboardInput(aboutFacilityUnissued.coverEndDateMonth().clear(), dateConstants.todayMonth);
+      cy.keyboardInput(aboutFacilityUnissued.coverEndDateYear().clear(), dateConstants.todayYear);
```

Comment:
Need to remove `.clear()` from all instances of `cy.keyboardInput` in this test (currently failing), e.g should be: 

```js
cy.keyboardInput(aboutFacilityUnissued.coverEndDateYear(), dateConstants.todayYear);
```

Reply:
Good spot, have done 👍